Formula/vim@7.4.rb

```diff
@@ -77,9 +77,18 @@ def install
       opts << "--without-x"
     end
 
-    if build.with? "luajit"
```

Comment:
I'm not sure who wins anything if the default position on versioned formulae is to let them rot, bugs and all. Applying the improved test is beneficial for Homebrew when this rolls through CI at any point in time, and applying this fix potentially stops users ending up with confusing linkage locally. It's not like either of these things is a nothing issue.